src/main/java/io/levchugov/petproject/controller/MovieController.java

```diff
@@ -0,0 +1,27 @@
+package io.levchugov.petproject.controller;
+
+import io.levchugov.petproject.model.Movie;
+import io.levchugov.petproject.repository.MovieJdbcRepository;
+import io.levchugov.petproject.service.MovieService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class MovieController {
+    private final MovieService movieService;
+
+    @GetMapping("/movies")
+    public ResponseEntity<List<Movie>> getMovieList (
+            @RequestParam(value = "chatId") Long chatId,
+            @RequestParam(value = "page") int page,
+            @RequestParam(value = "unitsOnPage") int unitsOnPage
```

Comment:
```suggestion
            @RequestParam(value = "limit") int limit
```
либо 
```suggestion
            @RequestParam(value = "size") int size
```

короче, а смысл тот же
для апи с пагинацией более привычные названия для такого параметра